diff --git a/node_modules/dooboo-ui/Icon/index.js b/node_modules/dooboo-ui/Icon/index.js
index 34fb1f7..34c9d92 100644
--- a/node_modules/dooboo-ui/Icon/index.js
+++ b/node_modules/dooboo-ui/Icon/index.js
@@ -1,7 +1,7 @@
 import collectingFontIconSelection from './selection.json';
 import { createIconSetFromIcoMoon } from 'react-native-vector-icons';
 import styled from '@emotion/native';
-const Ico = createIconSetFromIcoMoon(collectingFontIconSelection, 'IcoMoon', 'doobooui.ttf');
+const Ico = createIconSetFromIcoMoon(collectingFontIconSelection);
 export const Icon = styled(Ico) `
   color: ${({ theme, color }) => color || theme.text};
 `;
